assets/javascripts/discourse/initializers/add-search-results.js

```diff
@@ -107,7 +108,7 @@ export default {
 
   initialize(container) {
     const currentUser = container.lookup("current-user:main");
-    if (getEncryptionStatus(currentUser) !== ENCRYPT_ACTIVE) {
+    if (!isTesting() && getEncryptionStatus(currentUser) !== ENCRYPT_ACTIVE) {
```

Comment:
I think we can avoid doing these checks in each initializer, but I can try to do that in another PR.